dev-lang/rust/rust-1.74.1-r101.ebuild

```diff
@@ -648,13 +655,59 @@ mrustc_bootstrap() {
 	# These flags are used in every invocation of our bootstrap `cargo`.
 	local cargo_flags="--target ${CFG_COMPILER_HOST_TRIPLE} -j $(makeopts_jobs) --release --verbose"
 
+	# mrustc requires gcc, so disable libcxx
+	[[ "${LLVM_USE_LIBCXX}" == "1" ]] && unset LLVM_USE_LIBCXX
```

Comment:
This is set in profiles but users might have set `-stdlib=libc++` themselves. See my other comment for how to handle this instead.